apps/web/src/pages/Teams/ViewTeamPage/components/ViewTeam.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { useIntl } from "react-intl";
+import { Team } from "~/api/generated";
+import { Input, TextArea } from "@common/components/form";
+import Pill from "@common/components/Pill/Pill";
+import useLocale from "~/../../../frontend/common/src/hooks/useLocale";
+
+interface ViewTeamProps {
+  team: Team;
+}
+
+export const ViewTeam = ({ team }: ViewTeamProps) => {
+  const intl = useIntl();
+  const { locale } = useLocale();
+  const form = useForm();
+
+  const departmentsPillsArray =
+    team?.departments && team.departments.length > 0
+      ? team.departments.map((department) => {
+          return (
+            <Pill color="primary" mode="outline" key={department?.id}>
+              {department?.name?.[locale]}
+            </Pill>
+          );
+        })
+      : null;
+
+  return (
+    <FormProvider {...form}>
```

Comment:
Huh. I thought it was a bit odd to put this in a form, but... this _is_ what we're doing on the ViewPool page, so... alright 🤷‍♂️ .

Reply:
Actually can we revisit this strategy? Not only does this make the page component much more complicated than necessary (we're instantiating a form provider and using `<Input>` components with a half-dozen attributes instead of a simple `<p>` tag), it's also misleading (and unnecessarily loud?) from an accessibility/semantic perspective. The page is full of input components, but every single one is read-only.
@esizer

Reply:
Agreed, avoiding forms for data display would be ideal.

Reply:
feels like the design would need a revisit first?

Reply:
Would it? We could keep the same design and implement it with `<p>` elements with gray backgrounds and rounded corners.

Reply:
however other many pages will need to be de-formed will need to be in a new issue
as for this page, done over commits https://github.com/GCTC-NTGC/gc-digital-talent/pull/5706/commits/f8995a054b0f0731b80bc242be9cf3d95588a996 and https://github.com/GCTC-NTGC/gc-digital-talent/pull/5706/commits/acde4abe685958890c2714e6de4c01b453df5fc9

Reply:
![image](https://user-images.githubusercontent.com/40485260/219489253-1cdc0414-c7ae-4290-8635-ff25801cea0b.png)